### DISCUSSION TITLE
Is there a way to pull charge usage names?

### DISCUSSION THREAD

**Discussion by Xalvas** [CJNE/ha-myenergi#511] [Ideas]
Hi there.
On the myenergi web portal, you can see charge session data, and if set up, it will display which user initiated the charge.

![image](https://github.com/CJNE/ha-myenergi/assets/78328220/75aded6f-c00b-4ef2-8151-ea14c8f5d294)

Is it possible to pull this "user" data into the integration. For example a latest charge entry done by _this user_?

Very new to this so apologies if I've missed any format.

Thanks

**Comment by G6EJD** [comment]
This isn’t a code issue, it’s more of a request and should really be covered by the comment process.

**Reply by Xalvas** [reply]
Apologies. I'm not sure what you mean? Have I put this in the wrong place?

I chose the discussions section rather than the issues section as I know it's not a problem, more just an idea?

Thanks
